tests/store/tracking/test_sqlalchemy_store.py

```diff
@@ -913,8 +914,8 @@ def test_log_metric(self):
 
         tkey = "blahmetric"
         tval = 100.0
-        metric = entities.Metric(tkey, tval, int(1000 * time.time()), 0)
-        metric2 = entities.Metric(tkey, tval, int(1000 * time.time()) + 2, 0)
+        metric = entities.Metric(tkey, tval, get_current_time_millis(), 0)
```

Comment:
Can we make the `timestamp` with automatically generated value of get_current_time_millis() if caller does not fill the argument ?

Reply:
We can do it in following PR if needed.

Reply:
> Can we make the timestamp with automatically generated value of get_current_time_millis() if caller does not fill the argument ?

Why?

Reply:
I personally prefer positional to avoid specifying timestamp and it defaults to an undesired value.

Reply:
> What's the motivation of this change?

In most cases, `client.log_metric(Metric(...))` we should fill the current time as timestamp. Automatically fill timestamp can prevent user to fill number with wrong unit.